### DISCUSSION TITLE
Github GraphQL API: discussion's first comment seems not to be parsed

### DISCUSSION THREAD

**Discussion by MyNameIsNeXTSTEP** [community/community#102940] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

When testing Github GraphQL API for accessing my discussion in the repositiry I found out that the first comment is absent.
Tried then to edit and update the discussion, same thing.

<img width="1510" alt="Screenshot 2024-02-01 at 02 59 31" src="https://github.com/community/community/assets/79173403/724126c1-1706-4718-b17c-f00f66f239c1">

<img width="1512" alt="Screenshot 2024-02-01 at 02 59 39" src="https://github.com/community/community/assets/79173403/6955ca22-e827-4a67-ae6a-d9bbe041c926">

**Comment by dkt147** [comment]
If you're encountering issues with accessing the first comment or updating discussions using the GitHub GraphQL API, you can follow these steps to troubleshoot and potentially resolve the issue:

1. Check Permissions:
   Ensure that the API token you are using has the necessary permissions to read and update discussions. Verify that your token has the `public_repo` or `repo` scope for public or private repositories, respectively.

2. Review the GraphQL Query/Mutation:
   Double-check your GraphQL query or mutation to make sure it includes the necessary fields and parameters. Ensure that you are correctly identifying the discussion and the first comment.

3. Use the REST API for Comparison:
   If you are having trouble with the GraphQL API, try using the corresponding REST API endpoints to perform similar operations. This can help determine if the issue is specific to the GraphQL API or if it's a broader problem.

4. Check GitHub API Documentation:
   Refer to the official GitHub GraphQL API documentation to confirm that you are using the correct query or mutation. GitHub regularly updates its API, and it's essential to align your requests with the latest documentation.

5. Test with a Different Discussion:
   Create a new discussion in the repository and try to access or update it using the GraphQL API. This can help determine whether the issue is specific to a particular discussion or more general.

6. Review GitHub Repository Settings:
   Check the settings of your GitHub repository, especially any settings related to discussions or issues. Ensure that discussions are enabled and configured correctly.

7. Inspect API Responses:
   Examine the responses you receive from the GraphQL API. GitHub may provide error messages or additional information that can help you identify the root cause of the issue.

8. Contact GitHub Support:
   If you have exhausted all options and still cannot resolve the issue, consider reaching out to GitHub support. They may be able to provide insights or address potential bugs or limitations in the API.

9. Community Forums:
   Visit GitHub's community forums or relevant discussion platforms to see if others have experienced similar issues. Community members or GitHub staff might provide helpful insights or solutions.

10. Consider Rate Limiting:
    Check if you are hitting any rate limits imposed by GitHub. If you exceed the limits, GitHub may temporarily block your API requests.

By systematically going through these steps, you should be able to identify and resolve the issue you're facing with accessing or updating discussions through the GitHub GraphQL API.
